generator/uscore/uscore_generator.rb

```diff
@@ -759,7 +816,7 @@ def get_search_param_hash(search_parameters, sequence, grab_first_value = false)
         search_parameters.each_with_object({}) do |param, params|
           params[param] =
             if param == 'patient'
-              '@instance.patient_id'
+              'patient'
```

Comment:
What's this change?

Reply:
Hah, we should just remove it.  I didn't look too closely at the logic, was doing a replace any time we are doing `@instance.patient_id` because they are now in a loop block where `patient` is defined.